### DISCUSSION TITLE
Option for large icons

### DISCUSSION THREAD

**Discussion by w-w-y** [BazzaCuda/CustomMenu#18] [Ideas]
This looks excellent, great job. There are a few of us in the audience that have vast experience. You know, _old_. Being able to use large icons, 32px, would be helpful.

**Comment by BazzaCuda** [comment]
As someone who's just had cataract surgery, I understand where you're coming from :D 

Sadly, this option would require a major reworking of **CustomMenu**, even if we were to assume that every 16x16 icon in a file comes with a 24x24 or 32x32 sibling.

As an alternative, I would suggest maybe experimenting with the Windows settings to display your desktop and apps at a larger setting.
e.g. these settings under "Display"...
![windows display settings](https://github.com/BazzaCuda/CustomMenu/assets/22550919/63c5e2f6-d606-4d27-b13b-88b0a21ebf6d)

**Reply by w-w-y** [reply]
Understood. Thanks for the quick reply.
